src/index.html

```diff
@@ -1,13 +1,292 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Miami</title>
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body class="page__body">
+    <header class="header">
+      <!-- <div class="container">
+        <div class="header__content">
+          <div class="header__panel">
+            <a href="#">
+              <img src="./images/Logo.png" alt="bose__logo">
+            </a>
+
+            <a
+              href="tel:+1 234 5555-55-55"
+              class="icon icon--phone">
+            </a>
+
+            <a
+            href="#menu"
+            class="icon icon--menu">
+            </a>
+          </div>
+
+          <div class="header__slogan">
+            <img src="./images/Vector 1.png" alt="sound__wave">
+          </div>
+          <p class="header__slogan-text"></p>
```

Comment:
The `header__slogan-text` seems to be empty. Please add appropriate content or remove it if not required 🧐